prow/config.yaml

```diff
@@ -593,8 +593,6 @@ presubmits:
           - name: pilot-codecov-token
             mountPath: /etc/codecov
             readOnly: true
-          - name: e2e-testing-kubeconfig
-            mountPath: /home/bootstrap/.kube
```

Comment:
You may want to keep this. Pilot needs the existence of this file to pass bazel build before calculate coverage.

Reply:
Code coverage should not need it
https://github.com/istio/pilot/pull/1254